<a href="https://colab.research.google.com/github/fid2445/Fids-Sales-Predictions/blob/main/Project_2_Mike_Fiddler.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import pandas as pd

In [4]:
nf = pd.read_excel('/content/drive/MyDrive/Netflix Project.xlsx')
nf.head()

id                                title   type  \
0  ts300399  Five Came Back: The Reference Films   SHOW   
1   tm84618                          Taxi Driver  MOVIE   
2  tm127384      Monty Python and the Holy Grail  MOVIE   
3   tm70993                        Life of Brian  MOVIE   
4  tm190788                         The Exorcist  MOVIE   

                                         description  release_year  \
0  This collection includes 12 World War II-era p...          1945   
1  A mentally unstable Vietnam War veteran works ...          1976   
2  King Arthur, accompanied by his squire, recrui...          1975   
3  Brian Cohen is an average young Jewish man, bu...          1979   
4  12-year-old Regan MacNeil begins to adapt an e...          1973   

  age_certification  runtime                 genres production_countries  \
0             TV-MA       48      ['documentation']               ['US']   
1                 R      113     ['crime', 'drama']               ['US']   
2                PG       91  ['comedy', 'fantasy']               ['GB']   
3                 R       94             ['comedy']               ['GB']   
4                 R      133             ['horror']               ['US']   

   seasons    imdb_id  imdb_score  imdb_votes  tmdb_popularity  tmdb_score  
0      1.0        NaN         NaN         NaN            0.600         NaN  
1      NaN  tt0075314         8.3    795222.0           27.612         8.2  
2      NaN  tt0071853         8.2    530877.0           18.216         7.8  
3      NaN  tt0079470         8.0    392419.0           17.505         7.8  
4      NaN  tt0070047         8.1    391942.0           95.337         7.7

Data source https://www.kaggle.com/datasets/victorsoeiro/netflix-tv-shows-and-movies

This data set shows us all the different movies and shows available on the app Netflix.  We have imdb ratings, runtime, genre, release year.  
I am looking into this data set as i want to see if people are favoring shows instead of movies these days meaning this is a classification problem.

In [6]:
nf.shape

(5806, 15)

Here we can see we have 15 features and 5806 rows.

With almost 6000 rows I suspect there will alot of cleaning needed before any predictions can be made.  Just in the header i can see missing values that will need addressing.

In [9]:
df = pd.read_excel('/content/drive/MyDrive/Loan Project.xlsx')
df.head()

Loan_ID Gender Married Dependents     Education Self_Employed  \
0  LP001002   Male      No          0      Graduate            No   
1  LP001003   Male     Yes          1      Graduate            No   
2  LP001005   Male     Yes          0      Graduate           Yes   
3  LP001006   Male     Yes          0  Not Graduate            No   
4  LP001008   Male      No          0      Graduate            No   

   ApplicantIncome  CoapplicantIncome  LoanAmount  Loan_Amount_Term  \
0             5849                0.0         NaN             360.0   
1             4583             1508.0       128.0             360.0   
2             3000                0.0        66.0             360.0   
3             2583             2358.0       120.0             360.0   
4             6000                0.0       141.0             360.0   

   Credit_History Property_Area Loan_Status  
0             1.0         Urban           Y  
1             1.0         Rural           N  
2             1.0         Urban           Y  
3             1.0         Urban           Y  
4             1.0         Urban           Y

Data source https://www.kaggle.com/datasets/ninzaami/loan-predication

This data set shows us the typical questions you get when applying for a loan.
Building a model to predict if you can get a loan or not is a classification problem. 

In [10]:
df.shape

(614, 13)

13 features by 614 rows, and i can already see missing data in this set so going to have to spend alot of time cleaning the data.  I think this set would be easier to work through as the numbers are more straight forward and we have quite a few yes or no categories. 